src/NuGet.Core/NuGet.ProjectModel/PackageSpecReferenceDependencyProvider.cs

```diff
@@ -311,7 +311,7 @@ private List<LibraryDependency> GetDependenciesFromSpecRestoreMetadata(PackageSp
             return dependencies;
         }
 
-        private static List<LibraryDependency> GetSpecDependencies(
+        internal static List<LibraryDependency> GetSpecDependencies(
```

Comment:
To honor @zivkan's comment about fake dependencies, there'd probably be a different call to get the central dependencies.

Reply:
Same thing applies